devops/prod/values.yaml

```diff
@@ -171,3 +171,6 @@ iam-cache-server-helm:
+      IPFS_CLUSTER_ROOT: AgCWuoYg0khlbDn1YokBmyhKQ/EZnAU8M/v4ry69ZVqS/gVsBNayYpwsjwoxE59f+xwSS7GC5eDJnyYdlc/XKS6NpTV1JdTRsDi1kQrC17uA2TaxGj56zoRdwpudfgPae+M1AQmb3IYhHwuTUeP0ON1eVRKudlb0PrIO+Stfrua61QbBEyn3yFYvt1/yBghU8Hbm292wzimd3jjnfC5gCb/E5VQne1ey9WMRsIe1HzfRJLxPk1j+vNZ1DhrixkjePttYp9h1CcLpY7EqFmfVJs8cKPpidPoSs3RxT8a+bpB+iVgDL8Ss8rYnOFETJLHJCol6jzd8rag1odahVRTYOgK44cXDQbuK5s+JzAa7IR4Dqy7AMG5wPO9kPAy/7TduL6ggx6CAAh+FQJmeSWFc5Wtg14U3sOvnn75DzHH3d0PkFdRx0g7B9lYdx3AZQySLByMR8BJ9PTFACMfLDhiolaCvlAVRTbnIObnwmUvRfhFIB4NTl+WI586u8oOjYtUnqC+CrGUdv5fhB/YMOmJb2Fp0olqybFKND1bgMxIU2IBqCakk3tinI6KS3P8nF16Kxu8v9StlE8o5HwLOIk/zFzGF89GJuFnRm15OU/+aUK0Ol1CHCQXwpjwKS7rah1d17n7rJqSFd6304mgO/NW2PH/8OHfdi9YNGA8ZNymYToBo22C9cE28RzstZcYPJ/XMBfbwr1lC8xxe+Sa7pt4rNMJITfoPxUPKfcqNcS++goRHGuA3AOqXSM0Hu6J67g==
+      IPFS_CLUSTER_USER: AgATQa+MNEwulYrwCGKRo0HIztQ/3Ge0yFEDCXkEWvjIVsDlkI1PglaA7JpRLK94abtgSnnnnH638m0cUH6pcsYWhqQIr29eRtxjHdwfnpFW/rRTZwbD1wWa9ZGoaFXeHBk9WFRWwA9WZ+ivN9MBH8Nb7UUCn74d/qVfneLSQB9sO/+5mEuKiTKiFAcGmuXj8DRsHqq+ieinKoZDWBUyYkiBG2ZLEcf95/MMj43uW7Pk5/lKjA1qjgFrMKjzUeYj3Nb/US9ifnrHSk8AxoACSZilQGvk1qCWfRIvI33wsTvTEtbIlaZ0W8uNWg5QYCtZrbRQiXleNj83qnfYveET/sk1yY+ZpoaBvK4srAVgXHKPamLsUIdwbf58CGLlRSGs4R19QLGhzWQNF28Sz8JLTCWe6GYewngwFl685jw/M6V1xOM9+gVy2+ataTZBAFkk7woMqNw9D50uJxnrHbK+JXNJNI4aQT6u13oT/ZZqc3Ssk3cDO4WNQoFPiWVGNjaUwegqT9qvKE3OVohtyot/0BXauTcnArgl5ZTdYeeSrC/ftseFnjMfwjtDLxaSsajilyBnY392XfZJtA05T70QmpdyBTv+2Bcep/lkSULuUWs8GY2BENNpVRFag6c5gyeNdzA/WfU4EMhP8dAArSLDQxNioVuTWLGmxWL2Rvop5IUX4vFC4W3F0CRUKDzX/9mqc1AWAC32ZPaWZf+aBfFC5A==
+      IPFS_CLUSTER_PASSWORD: AgBHwmoO6LJ3hCAYMFxjeZyDdsAERdupnUHZs87nV3PRjIQ2ZQRCZk+6MrpgoReNpyu6cteefF3rGrgW2GHqSmgvfKOGzIPwQRCj5XLaW/dKf1dtwFKFzyFBGOfeZ2QkKzgMXz2APdulPoUAmAlri7p/B7lp5CjbzJ3+Tyildc/JSFKwIL8gtRUEPdtIqzokmWp7HQ1P2NJfM6tas2KxC8JqrKDC8pqutft4C5RN1p8Grsyi2kW/oDUhwLDgl8oJJnhu7GvgelAugpyvWFzCw6KBXgYs5cydRsAZ1caQZbrgtZcKCi0mxbu0omqr4o0bMNuinUgpa9Ib7aBsH+dRHgAbyYPv/HL0jk9yuoA2mnKQ+bu8VaLZcwxRTfTqyreTlYYukEnYDTspXsj9DcdRf/hyfYLtYegaCnSID/cY+nSOLrqc0KdtxeGiJuc6msPYQI1qOjXVHPwFJSCEjb2Aph10lSNumzsUC9kMfXePi0lb1LRkdOuKQojI5oQfij06V+eHyaaJpsiRgh9ZNtOfZEoniYWPMfh/NwS/FewRvTD6l6Prr1ZVbv/d1mLoy+T0WAsRSPn5fpiUIGNvErRNEYM5g3WLlEmH/Zca+7eiFL+suR03pJdpylKF4JQ+MD15DKbGNtdEQHfJHG+AKlMRexMItkvWFTdL/bYgo4Xhf2bAFv+yhZ+T27ld8vgX8xg/VXRDbdzC62+kZS4cvW7GnMz6qDdbRg==
```

Comment:
@JGiter I don't think these can be the right values, as we don't even have the prod cluster up yet, right?

Reply:
right. I thought we will have one ipfs for all environments like when using Infura. Should I leave IPFS cluster config only in dev env?

Reply:
> Should I leave IPFS cluster config only in dev env?

Yes, I think so, as we don't have it set up for other environments